lib/charms/observability_libs/v0/juju_topology.py

```diff
@@ -1,301 +0,0 @@
-# Copyright 2022 Canonical Ltd.
```

Comment:
is this deleted intentionally?

Reply:
Yes, we haven't used the deprecated `juju_topology` anywhere.